app/controllers/v0/sessions_controller.rb

```diff
@@ -19,6 +19,12 @@ def new
       type = params[:signup] ? 'signup' : params[:type]
       if REDIRECT_URLS.include?(type)
         url = url_service.send("#{type}_url")
+
+        # If redirecting to ID.me, include GA clientId for cross-domain analytics
+        if %w[idme signup].include?(type)
```

Comment:
I don't think we need to have this condition. All of the different login types go first to id.me before id.me redirects it to DS logon or MHV.